unittests/AST/ASTImporterVisibilityTest.cpp

```diff
@@ -49,6 +53,9 @@ auto *AnonV = "namespace { extern int v; }";
 // CXXRecordDecl:
 auto *ExternC = "class X;";
 auto *AnonC = "namespace { class X; }";
+// EnumDecl:
```

Comment:
Could we have a test for C++11 enum classes too?
"enum class E;"

Reply:
This is another case, I can add it.

Reply:
The enums are currently always "merged" at import, this is probably not always correct. For example when an enum is declared as friend (but maybe this is not problem here?).

Reply:
Yes, that may not be a problem in case of Enums. Though, on the other hand it would be more consistent to not "merge" enum classes and handle them similarly to RecordDecls.

For now, maybe the best would be to skip the test with the definition and prototype but with a FIXME to indicated that enum decls are merged.

Reply:
The tests work with the new Xxx"WithDef" kind of tests (will be renamed to "XxxWithMerge"). So the tests can be added, maybe with FIXME (at the test or at the import code?).

Reply:
Still more fixes are needed: If there is an existing enum class fwd declaration the definition is not imported. Should this go into a new PR? (The various fixes of enum class import can be taken as one thing in one PR.)

Reply:
Okay, so let's pretend in this PR that enum classes are nonexistent. And then create a separate PR which fixes enum classes related errors.

Reply:
Could you accept it now?
The un-scoped enum is handled here.